### PR TITLE
Core - onDragEnd callback

### PR DESCRIPTION
**Pull request by cw-ajit**

### CODE REVIEW

File: src/PinchZoomPan.js [hunk 1]
```diff
@@ -166,8 +181,29 @@ export default class PinchZoomPan extends React.Component {
...
+        const { startX, endX, startY, endY, startTime, endTime, isMoved } = this.dragCoords;
+        const deltaX = endX - startX;
+        const deltaY = endY - startY;
+
+        const distance = Math.sqrt(deltaX * deltaX + deltaY * deltaY);
+        const timeDiff = endTime - startTime;
+        const velocity = distance / timeDiff;
+
+        let horizontalDragDirection = deltaX > 0 ? 'right' : 'left';
+        let verticalDragDirection = deltaY > 0 ? 'down' : 'up';
```
Comment (enayat-carwale) [inline src/PinchZoomPan.js:196]: Do we ever need vertical direction, velocity ?

File: src/PinchZoomPan.js [hunk 2]
```diff
@@ -80,12 +80,20 @@ export default class PinchZoomPan extends React.Component {
     ANIMATION_SPEED = this.props.animationSpeed;
     onClickTimeoutId = null;
     hasMovedImg = false;
+    dragCoords = {startX: null, startY: null, endX: null, endY: null, isMoved: false, startTime: null, endTime: null};
```
Comment (enayat-carwale) [inline src/PinchZoomPan.js:83]: check if isMoved Required or not we can use deltaX or deltaY

**Review by enayat-carwale — COMMENTED** [review]
Reviewed

File: src/PinchZoomPan.js (hunk 1 above)
Comment (cw-ajit) [reply src/PinchZoomPan.js:196]: In the future, we might need.

**Pull request merged**